_includes/v22.2/ui/active-statement-executions.md

```diff
@@ -0,0 +1,38 @@
+
+## Active Executions table
+
+Click <img src="{{ 'images/common/ui-columns-button.png' | relative_url }}" alt="Column selector" /> to select the columns to display in the table.
+
+Column | Description
+-----|------------
+Statement Execution ID | The execution ID of the statement execution.
+Statement Execution | The SQL statement that was executed.
+Status | The status of the execution: Preparing, Waiting, or Executing.
+Start Time | The time the execution started.
```

Comment:
Nit: I have used "timestamp" for UI values like this, since the term is more precise. It's up for debate/discussion, but raising this for consistency reasons.